_data/sites.json

```
@@ -19101,6 +19101,16 @@
         "difficulty": "hard",
         "domains": ["targetprocess.com"]
     },
+    {
+        "name": "Taskade",
+        "url": "https://www.taskade.com/settings",
+        "notes": "Go to Account ⮕ Scroll down to Delete Account and Enter your Email ID",
+        "difficulty": "easy",
+        "domains": [
+            "taskade.com",
+            "www.taskade.com"
+        ]
```

```suggestion
        "domains": ["taskade.com"]
```
No need for the version with leading www